name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/Extractor.java

```diff
@@ -676,6 +681,71 @@ public Status apply(ImportAction action, Context context)
         }
     }
 
+    static class IgnoredItem extends Item
```

Comment:
Should we add a "reason" label to the item?
The the importer could say *why* it ignored that item.
And this is what we could display as message in the dialog.